### PR TITLE
update readme_en

### PR DESCRIPTION
**Pull request by mls1999725**

### CODE REVIEW

File: README_en.md [hunk 1]
```diff
@@ -2,178 +2,264 @@ English | [简体中文](README_cn.md)
...
+            <li>CornerNet-Squeeze</li>
+            <li>FCOS</li>  
+            <li>TTFNet</li>
+          </ul>
+        </ul>
+        <ul>
+          <li><b>Instance Segmentation</b></li>
+            <ul>
+             <li>Mask RCNN</li>
+             <li>SOLOv2 is coming soon</li>
```
Comment (yghstill) [inline README_en.md:88]: delete `is coming soon`. Already release. By the way, please also correct the relevant content in the Chinese document

File: README_en.md [hunk 2]
```diff
@@ -2,178 +2,264 @@ English | [简体中文](README_cn.md)
...
-- [x] **Group Norm**
-- [x] **Modulated Deformable Convolution**
-- [x] **Deformable PSRoI Pooling**
-- [x] **Non-local and GCNet**
-
-**NOTE:** Synchronized batch normalization can only be used on multiple GPU devices, can not be used on CPU devices or single GPU device.
-
-The following is the relationship between COCO mAP and FPS on Tesla V100 of representative models of each architectures and backbones.
+### Product dynamic
+
```
Comment (yghstill) [inline README_en.md:18]: Add：
- 2020.10.01: Add SOLOv2 model for instance segmentation, which  reached 38.6 FPS on single Tesla V100, and mask ap on the COCO-val dataset reached 38.8, which increased inference speed by 24%, mAP increased by 2.4 percentage points.

please also replenish the relevant content in the Chinese document

File: README_en.md [hunk 3]
```diff
@@ -2,177 +2,263 @@ English | [简体中文](README_cn.md)
...
 
-[PP-YOLO](https://arxiv.org/abs/2007.12099), which is faster and has higer performance than YOLOv4,
-has been released, it reached mAP(0.5:0.95) as 45.2%(newest 45.9%) on COCO test2019 dataset and
-72.9 FPS on single Test V100. Please refer to [PP-YOLO](configs/ppyolo/README.md) for details.
+PaddleDetection implements varied mainstream object detection algorithms in modular design, and provides wealthy data augmentation methods, network components(such as backbones), loss functions, etc., and integrates abilities of model compression and cross-platform high-performance deployment. 
 
-**Now all models in PaddleDetection require PaddlePaddle version 1.8 or higher, or suitable develop version.**
+After a long time of industry practice polishing, PaddleDetection has had smooth and excellent use experience, it has been widely used by developers in more than ten industries such as industrial quality inspection, remote sensing image object detection, automatic inspection, new retail, Internet, and scientific research.
```
Comment (qingqing01) [inline README_en.md:11]: use experience -> user experience

File: README_en.md [hunk 4]
```diff
@@ -2,177 +2,263 @@ English | [简体中文](README_cn.md)
...
+  - [Pedestrian detection](docs/featured_model/CONTRIB_cn.md)
+  - [Vehicle detection](docs/featured_model/CONTRIB_cn.md)
+- Competition Plan
+  - [Objects365 2019 Challenge champion model](docs/featured_model/champion_model/CACascadeRCNN.md)
+  - [Best single model of Open Images 2019-Object Detction](docs/featured_model/champion_model/OIDV5_BASELINE_MODEL.md)
+
+
+## Updates
+
+v0.4.0 was released at `07/2020`, add PP-YOLO, TTFNet, HTC, ACFPN, etc. And add BlaceFace face landmark detection model, add a series of optimized SSDLite models on mobile side, add data augmentations GridMask and RandomErasing, add Matrix NMS and EMA training, and improved ease of use, fix many known bugs, etc. Please refer to [版本更新文档](docs/CHANGELOG.md) for details.
```
Comment (qingqing01) [inline README_en.md:255]: 版本更新文档 -> change log

Comment (mls1999725) [reply README_en.md]: Done, thanks

File: README_en.md (hunk 3 above)
Comment (mls1999725) [reply README_en.md]: Done, thanks

File: README_en.md (hunk 1 above)
Comment (mls1999725) [reply README_en.md]: Done and so did the Chinese document, thanks

File: README_en.md (hunk 2 above)
Comment (mls1999725) [reply README_en.md]: Done and so did the Chinese document, thanks

**Pull request closed without merging**